command/agent/agent.go

```diff
@@ -215,6 +215,7 @@ func (c *cmd) run(args []string) int {
 		config.SerfPortLAN, config.SerfPortWAN))
 	ui.Info(fmt.Sprintf("Gossip Encryption: %t", config.EncryptKey != ""))
 	ui.Info(fmt.Sprintf(" Auto-Encrypt-TLS: %t", config.AutoEncryptTLS || config.AutoEncryptAllowTLS))
+	ui.Info(fmt.Sprintf("     ACLs Enabled: %t", config.ACLsEnabled))
```

Comment:
It's critical to also include what the ACL default policy is, not just that the ACL system is enabled.

If the ACL system is enabled, but the default policy was never set to "deny" (the default is "allow"), then that's effectively the same as having the ACL system disabled.

@cizh : Copying you to help think through how to communicate this best to the user.

For example, I'm worried that if the display shows:
```
ACLs Enabled: true
ACL Default Policy: allow
```

... they might think that things are set up securely after viewing the "ACLs Enabled" line, and maybe glaze over the default policy line that follows.

Reply:
Maybe it could be something like:

```
Secure ACLs: false (default policy is allow)
Secure ACLs: false (ACL system disabled; default policy is allow)
Secure ACLs: true (ACL system enabled with default policy of deny)
```

Reply:
Good call. I will mark this as a draft until we know what the message should look like.

Reply:
I also noticed that the startup output in the consul agent docs is incomplete and can update that as part of this PR as well.

https://developer.hashicorp.com/consul/docs/agent#understanding-the-agent-startup-output

Missing items:
* `Version`
* `Build Date`
* `Node ID`
* `Gossip Encryption`
* `Auto-Encrypt-TLS`
* `ACLs Enabled` <-- whatever fields we add for this one
* `HTTPS TLS`
* `gRPC TLS`
* `Internal RPC TLS`

Reply:
After discussing, @cizh and I recommend moving forward with:
```
ACLs Enabled: true
ACL Default Policy: allow
```

Reply:
Updated in [ccc8de7](https://github.com/hashicorp/consul/pull/17086/commits/ccc8de7c7e7d5a18492896f4ac64329ebe42df75)